src/main/java/htsjdk/variant/variantcontext/VariantContext.java

```diff
@@ -1682,6 +1682,17 @@ public int getEnd() {
         return (int)stop;
     }
 
+    /**
+     *
+     * @return whether the variant context meets the requirements to be a REF-only block or not
+     *
+     */
+    public boolean isRefOnlyBlock(){
```

Comment:
Should you also check that the genotypes are HomRef calls?

Reply:
How would you get non homref genotypes for a ref block?  (I understand very little about this topic)

Reply:
maybe rename this to `hasSpecificNonRefAllele` and negate the usage of said methods

Reply:
the check for `END` wouldn't make sense in method named `hasSpecificNonRefAllele` and I do want to check for that.